src/main/webapp/history/history.js

```diff
@@ -19,51 +18,337 @@
                 tooltip.enable();
             });
         });
+		
+		function renderTrendChart(chartDivId, model, settingsDialogId, chartClickedEventHandler) {
+			const chartPlaceHolder = document.getElementById(chartDivId);
+			const chart = echarts.init(chartPlaceHolder);
+			chartPlaceHolder.echart = chart;
+			
+			const textColor = getComputedStyle(document.body).getPropertyValue('--darkreader-text--text-color') || getComputedStyle(document.body).getPropertyValue('--text-color') || '#222';
+			const showSettings = document.getElementById(settingsDialogId);
+			let darkMode = getComputedStyle(document.body).getPropertyValue('--darkreader-bg--background')
+			darkMode = darkMode !== undefined && darkMode !== null && darkMode !== ''
+			console.log('darkMode: ' + darkMode)
+			let series = model.duration.series.concat(model.result.series)
+			series.forEach(s => s.emphasis = {
+				disabled: true
+			});
+			const options = {
+				animation: false,
+				darkMode: darkMode,
+				//backgroundColor: getComputedStyle(document.body).getPropertyValue('--darkreader-bg--background') || getComputedStyle(document.body).getPropertyValue('--bs-body-bg') || '#fff',
+				toolbox: {
+					feature: {
+					  dataZoom: {
+						yAxisIndex: 'none'
+					  },
+					  restore: {},
+					  saveAsImage: {}
+					}
+				},
+				tooltip: {
```

Comment:
not sure where exactly the code is but now that history is available when there is only one result the tool box is rendering NaN for a couple of fields:

<img width="173" alt="image" src="https://github.com/jenkinsci/junit-plugin/assets/21194782/ccd07bcf-3994-49cc-97f3-b87ecc62b655">

This should either be hidden or set to some value I think, users shouldn't see NaN